src/app/browser/qgsinbuiltdataitemproviders.cpp

```diff
@@ -290,7 +290,7 @@ void QgsAppDirectoryItemGuiProvider::populateContextMenu( QgsDataItem *item, QMe
   static int MAX_HIDDEN_ENTRIES = 5;
   for ( const QString &path : hiddenPathList )
   {
-    QAction *action = new QAction( QDir::toNativeSeparators( path ), hiddenMenu );
+    QAction *action = new QAction( QDir::toNativeSeparators( QString( path ).replace( QChar( '|' ), QChar( '/' ) ) ), hiddenMenu );
```

Comment:
Do you mind adding some comments explaining why the | characters may be there?

Reply:
I can, but I don't really know why the '/' are replaced with '|'  in the first place: **Not my Code (TM)**